site/content/comparison.md

```diff
@@ -13,3 +13,4 @@ REUSE does not intend to replace, but rather complement them. We try to solve un
 * [**ClearlyDefined**](https://clearlydefined.io) collects and displays meta and security information about a large number of projects distributed on different package registries. It also motivates developers and curators to extend data about a project's licensing and copyright situation. REUSE in comparison concentrates on fixing the problem at the file level for individual projects, which in turn will ease ClearlyDefined's efforts.
 * [**OpenChain**](https://www.openchainproject.org) focuses on making Free Software license compliance more transparent, predictable, and understandable for participants in the software supply chain. OpenChain recommends REUSE as one component to increase clarity of the licensing and copyright situation, but has higher requirements to achieve full conformance.
 * [**FOSSology**](https://www.fossology.org) is a toolkit for Free Software compliance, stores information in a database, and includes license, copyright and export scanners. It is more complex than REUSE and its helper tool and rather optimized for compliance officers and lawyers. REUSE instead intends to have all licensing and copyright information stored in or next to the source files to safeguard this information when reused elsewhere.
+* [**ORT**](http://oss-review-toolkit.org/), the OSS Review Toolkit, is a set of tools that complement existing compliance projects. It has the ability to analyze transitive dependencies declared by various package managers, and can download the source code for all dependencies in order to get it scanned by the license scanner of your choice. Furthermore, ORT is able to evaluate license findings against customizable policy rules, and it can create different reports like disclosure documents or Bill-Of-Materials (BOMs) in multiple formats.
```

Comment:
Full ack on the first sentence. But I rather miss a comparison with REUSE, which is what this page is about.

What do you think about replacing the last sentence (which rather is a feature list) by something like:

"With a scanned project and its dependencies being REUSE compliant, it will be much easier for ORT to correctly determine the licenses of all files used in the project."

Reply:
While that sounds nice, it's unfortunately not necessarily correct. Unless the license scanner that ORT is configured to use actually understands REUSE's specialties (i.e. the contents of the `LICENSES` and / or `.reuse` directories), there is no benefit for ORT if a project is REUSE-compliant or not. A "dumb" scanner simply does not understand to which file REUSE associates a license.

IMO it's really hard to compare REUSE and ORT here, as I regard REUSE mostly as a *standard* (and a tool to check for the standard), whereas ORT is ("only") a *tool*.

But I'll try to think about a better wording over the next days.

Reply:
Guess what @mxmehl, I finally realized this PR is still unfinished 😬 Please have another look!

Reply:
Woah, congrats that you remembered this PR ;)

LGTM, thank you!